src/components/ToDoContainer/NewTodoForm.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const NewTodoForm = (props) => {
+    return (
+        <form onSubmit={props.formSubmitted}>
```

Comment:
Avoid using the form component - instead, just implement the state functionality yourself. 

1. Rewrite this component from an SFC (stateless functional component) to a class-based component (like the ToDoContainer)

2. Add a constructor with a default state of `{todo: ''}`

3. Write a function which updates this LOCAL state to the new value of the text input when the input is changed, and pass it to the `input` tag below as the `onChange` prop.

4. Write a function which adds the new todo to the higher state when the user clicks the submit button, and pass this in to the `button` tag as an `onClick` prop.